libs/model/src/models/community.ts

```diff
@@ -153,6 +153,10 @@ export default (
         defaultValue: false,
         allowNull: false,
       },
+      primary_token_address: {
```

Comment:
What is this? We need descriptive names for token addresses. We have clanker tokens, launchpad tokens, pinned tokens, stake tokens... We can't add random token addresses and expect to know what they are for.